### ISSUE TITLE
Unfriendly error message when now.json doesn’t follow schema

### ISSUE THREAD

**Issue by 4cm4k1** [vercel/vercel#1375]
```shell
> Error! An unexpected error occurred!
  Error: You have the following errors inside `now.json` [{"keyword":"additionalProperties","dataPath":"","schemaPath":"#/additionalProperties","params":{"additionalProperty":"description"},"message":"should NOT have additional properties"}]
    at validateNowJson (/snapshot/repo/dist/now.js:17432:17)
    at readMetaData (/snapshot/repo/dist/now.js:17292:5)
    at <anonymous> Error: You have the following errors inside `now.json` [{"keyword":"additionalProperties","dataPath":"","schemaPath":"#/additionalProperties","params":{"additionalProperty":"description"},"message":"should NOT have additional properties"}]
    at validateNowJson (/snapshot/repo/dist/now.js:17432:17)
    at readMetaData (/snapshot/repo/dist/now.js:17292:5)
    at <anonymous>
```

I gather from this that `description` is no longer a valid field in `now.json`, but it’s a little bit hard to parse this error message — actually, it looks like it’s intended to be displayed more prettily, but for some reason isn’t. 😅

This is related to https://github.com/zeit/now-cli/pull/1374.

**Comment by leo** [comment]
@4cm4k1 AFAIK, `description` never was a valid field in `now.json`, as per [the docs](https://zeit.co/docs/features/configuration). Should be displayed in a more pretty way, however, agreed!

**Comment by 4cm4k1** [comment]
Thanks, @leo! It’s likely true that `description` was never valid. That’s my bad, though when I first wrote my `now.json`, the docs weren’t really fleshed out yet. And it never previously caused an error. 😛 Just wanted to bring visibility to the issue.

Something like…
```shell
> Error! Your `now.json` should NOT have additional properties, such as `description`.
  More info on `now.json` schema: https://zeit.co/docs/features/configuration
```
…would be great! 😄
